framework/db/mysql/ColumnSchema.php

```diff
@@ -18,17 +18,6 @@
  */
 class ColumnSchema extends \yii\db\ColumnSchema
 {
-    /**
-     * @var bool whether the column schema should OMIT using JSON support feature.
-     * You can use this property to make upgrade to Yii 2.0.14 easier.
-     * Default to `false`, meaning JSON support is enabled.
-     *
-     * @since 2.0.14.1
-     * @deprecated Since 2.0.14.1 and will be removed in 2.1.
```

Comment:
@SilverFire How this is possible to add and deprecate property in the same release? :D

Reply:
Why not? :)

Reply:
Because we can :)

Reply:
![When you can](https://media.giphy.com/media/3rgXBvnbXtxwaWmhr2/giphy.gif)